public/index.html

```diff
@@ -35,6 +35,8 @@
 
       <button id="periodicbackgroundsync" type="button">Wallpaper</button>
 
+      <button id="notificationtriggers" type="button">Reminder</button>
```

Comment:
This should probably be a checkbox, so it can be turned on and off.

Reply:
Triggered notifications can be canceled as outlined [here](https://web.dev/notification-triggers/#canceling-a-scheduled-notification).